.github/workflows/build-and-test.yml

```
@@ -0,0 +1,47 @@
+on:
+  push:
+    branches: ["**"]
+  pull_request:
+    branches: ["**"]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  concurrency:
```

actionlint (1.7.7)

"steps" section is missing in job "concurrency"

(syntax-check)